.appveyor.yml

```diff
@@ -56,7 +56,6 @@ install:
 
     # install composer dependencies
     - cd C:\projects\migrations
-    - rm composer.lock
```

Comment:
This file was modified in a commit labeled "appveyor". Please rewrite your commit history and include an explanation about this in your commit message :pray:

Reply:
https://github.com/doctrine/migrations/commit/12d408e7caa5fb9f5e921152b8ee6cfd4cc72d64#diff-354f30a63fb0907d4ad57269548329e3R19 this added the idea of commiting the composer.lock file in the repo, and from my understanding, it is a very bad idea.

Some reasons:
1) tests are run with outdated deps (as example some CS failures were not discovered because of it)
2) devs when contributing, and running composer install on their computers might get dependencies not compatible with their dev environment


The only reason I see to commit the composer.lock is to run tests faster... was it.

@jwage @alcaeus I can put it back, but should be clear which PHP version are we targeting

Reply:
I don't like that policy either, but the actual reason is documented here: https://www.doctrine-project.org/policies.html

I think we shouldn't remove the `composer.lock` from version control without amending or removing that policy page beforehand.

Reply:
reverted in https://github.com/doctrine/migrations/pull/858/commits/4eda5e6f78b2bc620adb56fea2423cfe47d3c1f1